### PR TITLE
feat: add orientation param #62

### PR DESCRIPTION
**Pull request by CAMOBAP**
- #62

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,27 +35,28 @@ Also, please note the following special message strings that can be returned via
...
 
-By default, this value is 120 seconds. Thus, you would want code similar to the following in your app when handling `onMessage` responses that return a passcode:
+By default, this value is 120 seconds. So, an `expired` error will be emitted to `onMessage` if you haven't called `event.markUsed()`.
 
-```
-this.timeoutCheck = setTimeout(() => {
-   this.setPasscodeExpired();
-   }, 120000);
-```
+Once you've utilized hCaptcha's token, call `markUsed` on the event object in `onMessage`, like `event.markUsed()`.
```
Comment (e271828-) [inline README.md:55]:
```suggestion
Once you've utilized hCaptcha's token, call `markUsed` on the event object in `onMessage`, like `event.markUsed()`.
```

@CAMOBAP please add example code

File: README.md [hunk 2]
```diff
@@ -35,27 +35,28 @@ Also, please note the following special message strings that can be returned via
...
+
+### Handling errors and retry
+
+In case of an `error`, you can `reset` hCaptcha by calling `event.reset()`` to perform another attempt at verification.
```
Comment (e271828-) [inline README.md:59]:
```suggestion
If your app encounters an `error` event, you can reset the hCaptcha SDK flow by calling `event.reset()`` to perform another attempt at verification.
```

**Review by e271828- — COMMENTED** [review]
missing readme update for orientation

File: README.md (hunk 1 above)
Comment (CAMOBAP) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (CAMOBAP) [reply README.md]: Applied

**Comment by CAMOBAP** [comment]
> missing readme update for orientation

Done, ready for review again

**Review by e271828- — APPROVED** [review]
please bump version as well.

**Pull request merged**